pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -61,7 +64,7 @@ func (c *Reconciler) createAffinityAssistants(wb []v1alpha1.WorkspaceBinding, pr
 					errs = append(errs, fmt.Errorf("failed to create StatefulSet %s: %s", affinityAssistantName, err))
 				}
 				if err == nil {
-					c.Logger.Infof("Created StatefulSet %s in namespace %s", affinityAssistantName, namespace)
+					logger.Infof("Created StatefulSet %s in namespace %s", affinityAssistantName, namespace)
```

Comment:
hey @mattmoor does this mean the reconciler no longer has a logger object or just that we shouldn't use it?